.gitignore

```diff
@@ -11,6 +11,9 @@ cache
 deployments/localhost
 deployments/hardhat
 typechain-types
-
+deployments
 # IDEA
 .idea
+.env
+.env.staging
+.env.alfajores
```

Comment:
Shouldn't these be committed as static configuration?

Reply:
great question. Released .env now to be a static configuration.
The rest actually represent unencrypted secrets, however there equivalent .env.network.enc is commited to git.